tests/providers/openai/hooks/test_openai.py

```diff
@@ -16,48 +16,58 @@
 # under the License.
 from __future__ import annotations
 
-from unittest.mock import Mock, patch
+from unittest.mock import MagicMock, Mock, patch
 
 import pytest
+from openai import OpenAI
+from openai.types import CreateEmbeddingResponse, Embedding
 
 from airflow.providers.openai.hooks.openai import OpenAIHook
 
 
 @pytest.fixture
 def openai_hook():
-    with patch("airflow.providers.openai.hooks.openai.OpenAIHook._get_api_key"), patch(
-        "airflow.providers.openai.hooks.openai.OpenAIHook._get_api_base"
-    ) as _:
+    with patch("airflow.providers.openai.hooks.openai.OpenAI") as _:
```

Comment:
May I know why do we need `as _`?

Reply:
yes, it may not be needed, removed it. thank you!